android/gradle.properties

```diff
@@ -26,3 +26,4 @@ POM_DEVELOPER_ID=uber
 POM_DEVELOPER_NAME=Uber Technologies
 android.useAndroidX=true
 android.enableJetifier=true
+org.gradle.jvmargs=-Xmx4096m
```

Comment:
We were just squeaking by before or leaking somewhere?

Reply:
Squeaking by locally. 2nd run would pass due to cached tasks